cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -482,10 +481,8 @@ public Void call() throws Exception {
           context.setState(new ProgramState(ProgramStatus.INITIALIZING, null));
           if (mapReduce instanceof ProgramLifecycle) {
```

Comment:
If that is the case, how will user configure datasets to be used for input and output?